ui/components/ui/tabs/tabs.stories.js

```diff
@@ -4,46 +4,37 @@ import Tabs from './tabs.component';
 
```

Comment:
Adding disabled story to storybook and reformatting to use the latest storybook [CSF](https://storybook.js.org/docs/6/api/stories/csf) format